### PR TITLE
Added permission related changes to DTM user documentation

### PR DESCRIPTION
**Pull request by spau6**
Added permission related changes to DTM user documentation

### CODE REVIEW

**Comment by spau6** [comment]
These changes has to be ported back to 10.18 and 10.17

File: content/dtm/asset-hierarchy-bundle/overview.md [hunk 1]
```diff
@@ -17,6 +17,16 @@ If you already created assets, you find them listed in a hierarchical view on th
 
 ![assets-view](/images/dtm/assets/dtm-assets-view-assets.png)
 
+{{< c8y-admon-req >}}
+ROLES & PERMISSIONS in assets context:
```
Comment (BeateRixen) [inline content/dtm/asset-hierarchy-bundle/overview.md:21]:
```suggestion
ROLES & PERMISSIONS
```

File: content/dtm/asset-types-bundle/asset-types.md [hunk 2]
```diff
@@ -35,6 +35,12 @@ The total number of asset models is displayed on the bottom left of the page.
 
 Use the **Export asset models** and **Import asset models** feature to transfer asset models between tenants. This is useful for replicating the asset models in multiple tenants. By exporting and importing asset models, you can ensure consistency and reduce the time and effort required for manual configuration. See [To export asset models](#to-export-asset-models) and [To import asset models](#to-import-asset-models) on how to export or import asset models.
 
+{{< c8y-admon-req >}}
+ROLES & PERMISSIONS for asset models:
```
Comment (BeateRixen) [inline content/dtm/asset-types-bundle/asset-types.md:39]:
```suggestion
ROLES & PERMISSIONS
```

This is redundant as you mention that it is for asset models below

File: content/dtm/asset-types-bundle/property-library.md [hunk 3]
```diff
@@ -16,6 +16,12 @@ helpContent:
 ---
 Asset properties are the parameters that define an [asset model](/dtm/asset-types/#asset-types). One or more asset properties are used to define an asset model. Asset properties are created in the **Asset properties** page.
 
+{{< c8y-admon-req >}}
+ROLES & PERMISSIONS for asset properties:
```
Comment (BeateRixen) [inline content/dtm/asset-types-bundle/property-library.md:20]:
```suggestion
ROLES & PERMISSIONS
```

This is redundant as you mention that it is for asset properties below

File: content/dtm/localization.md [hunk 4]
```diff
@@ -22,6 +22,10 @@ helpcontent:
 ---
 
 {{< c8y-admon-req >}}
+* To view Localization page: READ permission for permission type "Application management"
```
Comment (BeateRixen) [inline content/dtm/localization.md:25]:
```suggestion
* To view the **Localization** page: READ permission for permission type "Application management"
```

File: content/dtm/localization.md [hunk 5]
```diff
@@ -22,6 +22,10 @@ helpcontent:
...
+
+* To add/update/delete Localization identifiers: ADMIN permission for permission type "Application management"
```
Comment (BeateRixen) [inline content/dtm/localization.md:27]:
```suggestion
* To add/update/delete localization identifiers: ADMIN permission for permission type "Application management"
```

**Review by BeateRixen — APPROVED** [review]

**Review by MWindel — APPROVED** [review]

**Pull request merged**